app/uk/gov/hmrc/individualsemploymentsapi/controller/DocumentationController.scala

```diff
@@ -32,7 +32,7 @@ class DocumentationController @Inject()(
   config: Configuration)
     extends BackendController(cc) {
   private lazy val privilegedWhitelistedApplicationIds =
-    config.getOptional[Seq[String]]("api.access.version-P1.0.whitelistedApplicationIds").getOrElse(Seq.empty)
+    config.getOptional[Seq[String]]("api.access.version-P2.0.whitelistedApplicationIds").getOrElse(Seq.empty)
```

Comment:
If we change this how does P1 documentation get served? HO will remain on V1 and so we need to keep the V1 thread. We need to add a new thread of docs for V2.

Reply:
The documentation itself is served up from the documentation method, which dynamically generates the path to the documentation files. Not sure what this line is used for (it's pointed to by the /api/definition route)...